torchvision/transforms/v2/_container.py

```diff
@@ -138,9 +138,7 @@ def __init__(
         if p is None:
             p = [1] * len(transforms)
         elif len(p) != len(transforms):
-            raise ValueError(
-                f"Length of p doesn't match the number of transforms: " f"{len(p)} != {len(transforms)}"
-            )
+            raise ValueError(f"Length of p doesn't match the number of transforms: " f"{len(p)} != {len(transforms)}")
```

Comment:
I think this is just pre-commit who did that

Reply:
Meaning, lint on `main` is red right now?

Reply:
Yup: https://app.circleci.com/jobs/github/pytorch/vision/1840273